cli/include/gz/utils/cli/GzFormatter.hpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GZ_UTILS_CLI_IGNITION_FORMATTER_HPP_
```

Comment:
Was this file moved with `git mv`? Its blame only shows the commits in this PR:

https://github.com/ignitionrobotics/ign-utils/blame/2c41344a782b5fe894493f34162620757ac5fadd/cli/include/gz/utils/cli/GzFormatter.hpp

It's similar for some of the other moved files

Reply:
yes, there's a problem with how the files are renamed. It should be a simple `git mv` command.

Compare https://github.com/ignitionrobotics/ign-utils/pull/53/commits/e55aa676f78fd9443f4f19ce27c9a9c5a5f236de with https://github.com/ignitionrobotics/ign-utils/pull/51/commits/b2e10989c84dc238af8a89d9768514d5dcc84b10

Reply:
Hmm, I think it's because to get the git diff to work in the script, I had to reset. I think I'll bypass that and force a commit, hopefully that gets the histories transferred!

Reply:
Fixed!

https://github.com/ignition-tooling/release-tools/pull/712/commits/0a46de18d5d7b44d317267f05fa83ef9e6c4d726

The issue was the script was ignoring staged changes when checking `git diff`. I edited the commit function to be more flexible (https://github.com/ignition-tooling/release-tools/pull/712/commits/daad715a0277002fc9426ec3e1fec0b256ac0fcb) to have it `git diff --staged` instead. (I was previously resetting/unstaging the changes which caused the blame to go awry.)

Reply:
The blame looks good to me now!

Another thing we should check is whether squash-merging this PR would preserve that, or if we need to rebase-merge.

Reply:
the header guard still has `IGNITION`

Reply:
Nice catch! Fixed!

https://github.com/ignition-tooling/release-tools/pull/712/commits/5a44a5a3edc1e2f2e1eeccb669b1418ab4a4a263

Reply:
the header guard still needs to be fixed in this file

Reply:
Whoops, looks like an edit to the regex I had on the script caused a reversion...

https://github.com/ignition-tooling/release-tools/commit/522cc9c3cce442f13cde90653db897e47b70c92f
https://github.com/ignitionrobotics/ign-utils/pull/55/commits/febac36a43e335f37c9a6e3b6479073e6a0a9119